core/modules/system/views/system.views.inc

```diff
@@ -120,6 +120,16 @@ function system_views_data() {
     ),
   );
 
+  // - info
+  $data['system']['info'] = array(
+    'title' => t('Module/Theme/Theme engine properties'),
```

Comment:
I agree that this should be called "Project properties". People are not able to use alternate theme engines in backdrop, so including that label here could be confusing for people. I don't think the omission of some project types is a problem, and would prefer this be consistent with other uses of "Project".

Reply:
@jenlampton I can see that this change would make things clearer. Please notice that there are a bunch of pre-existing views data in `system.views.inc` that already used "Module/Theme/Theme engine..." wording. I can make the change you request here, but do you also want me to change the rest of the data there to match?